pay-api/src/pay_api/models/payment_account.py

```diff
@@ -63,7 +63,7 @@ class Meta:  # pylint: disable=too-few-public-methods
         """Returns all the fields from the SQLAlchemy class."""
 
         model = PaymentAccount
-        exclude = ['versions', 'pad_activation_date', 'id']
+        exclude = ['versions', 'pad_activation_date']
```

Comment:
one of the tests needs the payment account id in the response ; 
Lmk if its okay to expose the id , If not i can find other ways to get the id...

Reply:
ok, id may confuse if it's included in the response as there would be both id and accountId in paymentAccount resource.

Reply:
will fix it in the next PR